src/main/java/com/xceptance/xlt/agentcontroller/FileManagerServlet.java

```diff
@@ -101,14 +101,12 @@ protected void doGet(final HttpServletRequest req, final HttpServletResponse res
                 final File file = new File(rootDirectory, fileName);
                 in = new FileInputStream(file);
 
-                resp.setContentLength((int) file.length());
-                // resp.setContentType("???");
-
+                resp.setStatus(HttpServletResponse.SC_OK);
+                resp.setHeader("Content-Length", String.valueOf(file.length()));
```

Comment:
I'd prefer using `resp.setContentLengthLong(file.length());` instead.

Reply:
Now that I know that this method exists, I'd prefer that too.   ;-)
Done.